Healthy/Classes/Generated/UIImage.Generated.swift

```diff
@@ -9,4 +9,12 @@ extension UIImage {
     static var accentColor: UIImage {
       UIImage(named: "AccentColor")!
     }
+
+    static var images: UIImage {
+      UIImage(named: "images")!
+    }
+
+    static var logoImg: UIImage {
+      UIImage(named: "logo-img")!
+    }
```

Comment:
Please rename the original image names in the assets to reflect the images description.